### PR TITLE
Fixing #6: toConstantCase

### PR DESCRIPTION
**Pull request by davidonlaptop**
Fixing #6: Adding toConstantCase method to convert a word into an uppercase only letters with words separated with underscores.

### CODE REVIEW

**Comment by martinandert** [comment]
Thanks for this, David, looking good. However I'm not so fond of the new method name, especially the `to` prefix. As an alternative to `constantize`, what do you think about changing the name to `constantify`? We already have `classify`...

File: test/cases.js [hunk 1]
```diff
@@ -302,5 +302,16 @@ module.exports = {
     'cow'    : 'kine',
     'zombie' : 'zombies',
     'genus'  : 'genera'
-  }
+  },
+
+  CamelToConstantCase: {
```
Comment (martinandert) [inline test/cases.js]: This needs a better name since only a few keys are camel-case.

File: test/index.js [hunk 2]
```diff
@@ -430,4 +430,10 @@ describe('Inflector', () => {
     assert.equal(Inflector.parameterize('Mädchen'), 'madchen');
     assert.equal(Inflector.parameterize('Mädchen', { locale: 'de' }), 'maedchen');
   });
+
+  it('properly converts camelCase to constant case', () => {
```
Comment (martinandert) [inline test/index.js]: Same here. What about just replacing `camelCase` with `strings`?

**Comment by davidonlaptop** [comment]
Sure, constantify would be more coherent!

File: test/index.js (hunk 2 above)
Comment (davidonlaptop) [reply test/index.js]: words ?

Comment (martinandert) [reply test/index.js]: :+1:

**Comment by davidonlaptop** [comment]
Hi @martinandert , the changes are completed and squashed :-)

**Comment by davidonlaptop** [comment]
Is there something else you would like to add to this feature?

**Comment by martinandert** [comment]
Thanks David, I just cut a new release with your changes.

**Comment by davidonlaptop** [comment]
Thanks Martin!

I've tried using the new release in my project, but the [typescript definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/inflected/index.d.ts) is not up to date yet.

Do you have plans on adding typescript bindings in your project?

**Comment by martinandert** [comment]
I don't know much about typescript (yet). Is that something that could/should be added to the project itself? If yes, feel free to open another PR.

**Comment by davidonlaptop** [comment]
Many projects add it directly in the repo, and that is considered best practice as you can evolve and version it with the code.

DefinitelyTyped is producing typescript bindings for hundreds of javascript libraries, which is better than nothing. I am using the one they have released for inflected 1.1.6, which works but miss the new function.

It's probably easy to do, but I have not written one before. The general idea is to create typed interfaces for the functions, parameters and return values exposed by the library. It's probably relatively easy to do.

However, I'm changing jobs in 2 weeks, so I can't guarantee that I'll have time to do it. I'll let you know if I have something that works.

**Pull request merged**